content/blog/wingingit/winging-it-10.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Learn CSS Grid First
+episode: 10
+sponsors: false
+tags:
+  - CSS
+  - Grids
+  - Layout
+author:
+  - miriam
+  - stacy
+date: 2024-07-23
+length: 54 mins
+image:
+  src: winging-it/winging-it-10.jpg
+media:
+  - youtube: aDMWD_CYpEI
+summary: |
+  We talk with Stephanie Eckles
+  – the brain behind ModernCSS.dev and SmolCSS.css
```

Comment:
Feels like we should link these, but if we can't/shouldn't then at minimum the address is SmolCSS.dev (not .css). I'll check to see if the YouTube description has this same typo? 

```suggestion
  – the brain behind [ModernCSS.dev](https://moderncss.dev/) and [SmolCSS.dev](https://smolcss.dev/)
```